tests/test_wikipron/__init__.py

```diff
@@ -17,7 +19,7 @@ def can_connect_to_wiktionary() -> bool:
         requests.get(
             "https://en.wiktionary.org/wiki/linguistics", headers=HTTP_HEADERS
         )
-    except (requests.ConnectionError, requests.ConnectTimeout):
+    except requests.ConnectionError:
```

Comment:
Out of curiosity, did you run into any issue with `requests.ConnectTimeout` here?

Reply:
I removed `ConnectTimeout` because it inherited from `ConnectionError` so didn't think it was necessary to also have the child class in the `except` clause. I've noticed that `ConnectTimeout` also inherits from the `Timeout` exception, so have added this line back in.